src/components/PeopleTable.tsx

```diff
@@ -1,52 +1,100 @@
-export const PeopleTable = () => {
+import { Link, useSearchParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../types';
+import { PersonLink } from './PersonLink';
+import { SortType } from './enums/SortType';
+import { SearchLink } from './SearchLink';
+
+type Props = {
+  people: Person[],
+  selectedSlug: string,
+};
+
+export const PeopleTable: React.FC<Props> = ({
+  people, selectedSlug,
+}) => {
+  const [searchParams] = useSearchParams();
+  const sort = searchParams.get('sort') || null;
+  const order = searchParams.get('order') || null;
+
+  const getSortType = (sortType: SortType) => (
+    {
+      sort: sort === sortType && order ? null : sortType,
+      order: sort === sortType && !order ? 'desc' : null,
+    }
+  );
```

Comment:
You should wrap this object in parenthesis to make it more readable and avoid potential issues with automatic semicolon insertion.